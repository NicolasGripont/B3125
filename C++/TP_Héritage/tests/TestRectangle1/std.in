R rect1 11 8 4 5
R rect1 1 2 3 4
R rect2 -5 -6 -1 0
HIT rect1 5 8
HIT rect1 10 9
HIT rect2 -5 -6
HIT rect2 -3 -2
DELETE rect1
R rect1 2 4 6 8
R rect3 2
R rect3 3 4 5 6 7 
EXIT