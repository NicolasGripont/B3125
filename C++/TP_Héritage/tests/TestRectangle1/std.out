OK
ERR
OK
YES
NO
YES
YES
OK
OK
ERR
ERR